addons/flux/playbook.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Setup flux
+  hosts: localhost
+  vars_files:
+    - "./ansible-vars.yaml"
+  environment:
+    KUBECONFIG: "{{playbook_dir}}/../../rke/kube_config_cluster.yml"
+  tasks:
+  - name: Install curl on debian or on ubuntu
+    apt:
+      name: "{{ item }}"
+      state: latest
+    with_items:
+    - curl
```

Comment:
we could add `[update_cache](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html#parameter-update_cache)`option to reload the cache in case, it is an outdated cache

Reply:
Ok done